Demo/src/main/java/com/braintreepayments/demo/MainActivity.java

```diff
@@ -284,6 +284,7 @@ private GooglePayRequest getGooglePayRequest() {
                 .setTotalPriceStatus(WalletConstants.TOTAL_PRICE_STATUS_FINAL)
                 .build());
         googlePayRequest.setEmailRequired(true);
+        googlePayRequest.setCountryCode("BR");
```

Comment:
Yeah in prod there's currently an error with our default allowed card networks (I think Maestro or some other one requires the country code to be BR) - updated this rather than changing allowed card networks

Reply:
Oh ok yeah that makes total sense 👍